docs/source/install/u20.rst

```diff
@@ -0,0 +1,209 @@
+``st2web`` and ``nginx`` packages, generate certificates or place your existing certificates under
+``/etc/ssl/st2``, and configure nginx with |st2|'s supplied :github_st2:`site config file st2.conf
+<conf/nginx/st2.conf>`.
+
+.. code-block:: bash
+
+  # Install st2web and nginx
+  sudo apt-get install -y st2web nginx
+
+  # Generate self-signed certificate or place your existing certificate under /etc/ssl/st2
+  sudo mkdir -p /etc/ssl/st2
+  sudo openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
+  -days XXX -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information \
+  Technology/CN=$(hostname)"
+
+  # Remove default site, if present
+  sudo rm /etc/nginx/conf.d/default.conf
+  # Check for a default site on sites-enabled to avoid a duplicate default server error 
+  sudo rm -f /etc/nginx/sites-enabled/default
+  # Copy and enable the supplied nginx config file
+  sudo cp /usr/share/doc/st2/conf/nginx/st2.conf /etc/nginx/conf.d/
+
+  sudo service nginx restart
+
+If you modify ports, or url paths in the nginx configuration, make the corresponding changes in
+the st2web configuration at ``/opt/stackstorm/static/webui/config.js``.
+
+Use your browser to connect to ``https://${ST2_HOSTNAME}`` and login to the WebUI.
+
+.. include:: common/api_access.rst
+
+Setup ChatOps
+-------------
+
+If you already run a Hubot instance, you can install the `hubot-stackstorm plugin
+<https://github.com/StackStorm/hubot-stackstorm>`_ and configure |st2| environment variables, as
+described below. Otherwise, the easiest way to enable :doc:`StackStorm ChatOps </chatops/index>`
+is to use the `st2chatops <https://github.com/stackstorm/st2chatops/>`_ package.
+
+* Validate that the ``chatops`` pack is installed, and a notification rule is enabled:
+
+  .. code-block:: bash
+
+    # Ensure chatops pack is in place
+    ls /opt/stackstorm/packs/chatops
+    # Create notification rule if not yet enabled
+    st2 rule get chatops.notify || st2 rule create /opt/stackstorm/packs/chatops/rules/notify_hubot.yaml
+
+* Add `NodeJS v10 repository <https://nodejs.org/en/download/package-manager/>`_:
+
+  .. code-block:: bash
+
+    curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+
+* Install the ``st2chatops`` package:
+
+  .. code-block:: bash
+
+    sudo apt-get install -y st2chatops
+
+.. include:: common/configure_chatops.rst
+
+* Start the service:
+
+  .. code-block:: bash
+
+    sudo service st2chatops start
+
+* Reload st2 packs to make sure ``chatops.notify`` rule is registered:
+
+  .. code-block:: bash
+
+    sudo st2ctl reload --register-all
+
+* That's it! Go to your Chat room and begin ChatOps-ing. Read more in the :doc:`/chatops/index` section.
+
+A Note on Security
+------------------
+
+.. include:: common/security_notes.rst
+
+Upgrade to |ewc|
+----------------
+
+.. include:: common/ewc_intro.rst
```

Comment:
```suggestion
```
Could be removed as EWC is no more

Reply:
This would leave the documentation inconsistent with the other distributions, if you're OK with that I'll include it in this PR, otherwise the EWC documentation should be addressed in a separate PR.

Reply:
I navigated to https://docs.stackstorm.com/install/u18.html#upgrade-to-ewc and it's just a disclaimer that EWC is open-sourced and available since st2 v3.4.

That looks good, thanks for the heads up :+1: